apps/prairielearn/src/tests/utils/group.ts

```diff
@@ -38,7 +38,8 @@ export async function switchUserAndLoadAssessment(
   const csrfTokenElement = $(form).find('input[name="__csrf_token"]');
   assert.nestedProperty(csrfTokenElement[0], 'attribs.value');
   assert.isString(csrfTokenElement.attr('value'));
-  const csrfToken = csrfTokenElement.attr('value') as string; // guaranteed to be string by assertion
+  const csrfToken = csrfTokenElement.attr('value');
+  assert.ok(csrfToken);
```

Comment:
Nit, the `assert.isString` assertion is now redundant:

```suggestion
  const csrfToken = csrfTokenElement.attr('value');
  assert.ok(csrfToken);
```